README.md

```diff
@@ -1,22 +1,41 @@
-# Welcome to Pocket Portal!
-
-### Run with Remix
+<div align="center">
+<h1>Grove Portal UI</h1>
+<img src="https://storage.googleapis.com/grove-brand-assets/Presskit/Logo%20Joined-2.png" alt="Grove logo" width="500"/>
+</div>
+<br/>
+
+# Table of Contents <!-- omit in toc -->
+
+- [Run with **Remix**](#run-with-remix)
+- [Deployment](#deployment)
+- [Development](#development)
+  - [Env](#env)
+  - [Node Version](#node-version)
+  - [Frontend](#frontend)
+  - [Stripe Webhook Forwarding](#stripe-webhook-forwarding)
+  - [Environment Variables](#environment-variables)
+  - [Backend](#backend)
+
+## Run with **Remix**
 
 - [Remix Docs](https://remix.run/docs)
 
 ## Deployment
 
-### PR / DEV
+1. **Test Locally**
+
+   - Test your changes locally before creating a PR.
 
-Create PR and vercel will create a preview build based on the PR
+2. **PR / STAGE**
 
-### STAGE
+   - Create a PR into the "staging" branch. Continuous Deployment (CD) will automatically deploy to [https://staging.portal.grove.city](https://staging.portal.grove.city).
 
-Merge PR into "stage" branch and vercel will build to staging environment
+3. **PROD**
 
-### PROD
+   - Create a PR from "staging" into "main". CD will automatically deploy to [https://portal.grove.city/](https://portal.grove.city).
```

Comment:
```suggestion
   - Create a PR from the `stage` branch into the `main` branch
   - CD will automatically deploy to [portal.grove.city](https://portal.grove.city).
   - E.g. <INSERT_EXAMPLE_LINK>
   
```

Reply:
Same question as above, I'n not following what the example link should be.

Reply:
This is a PR that merges `stage` into `main`: https://github.com/pokt-foundation/grove-portal/pull/662

You can use it here and find another one for the example above.

Reply:
Done.